packages/@aws-cdk/core/test/secret-value.test.ts

```diff
@@ -21,12 +21,12 @@ nodeunitShim({
     // WHEN
     const v = SecretValue.secretsManager('secret-id', {
       jsonField: 'json-key',
-      versionId: 'version-id',
+      versionId: '',
       versionStage: 'version-stage',
     });
```

Comment:
OK. Let's remove this line completely then:

```suggestion
    const v = SecretValue.secretsManager('secret-id', {
      jsonField: 'json-key',
      versionStage: 'version-stage', 
    });
```